lcls2-cc-lib/Library/Version/Global_Version.TcGVL

```diff
@@ -3,10 +3,10 @@
   <GVL Name="Global_Version" Id="{28d53d87-c928-4af1-84ef-e6ceb5302370}">
     <Declaration><![CDATA[{attribute 'TcGenerated'}
 {attribute 'no-analysis'}
+{attribute 'linkalways'}
```

Comment:
Not sure why this automatically changed but it changed for me too in the other PR :shrug: